src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -100,6 +101,14 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
     }));
   };
 
+  useEffect(() => {
+    const allNurseries = getAllNurseries(organization);
+    const destinantionNurseries = allNurseries.filter((nur) => nur.id.toString() !== selectedNursery);
```

Comment:
typo destinationNurseries
also we could rename `nur` to `nursery`? :) more readable..